kordex/build.gradle.kts

```diff
@@ -71,9 +69,3 @@ dependencies {
 tasks.withType<ShadowJar> {
     archiveClassifier = ""
 }
-
-publishing {
-    repositories {
-        mavenLocal()
```

Comment:
Why? This is how the test module works.

Reply:
See settings.gradle.kts

Reply:
Then the test module isn't a realistic test, and it can't be used to test old versions when needed. There was no need to change this block or the `repositories` block in the test module, IMO.

Reply:
It's common convention to use include build to test Gradle plugins as it doesnt require you to run anything before testing a change

Reply:
I'm not particularly concerned with common convention, to be blunt - if I was, KordEx would be considerably harder to read.

I'd prefer a realistic test case, and that's why I set things up how they were. Again, there was no _need_ to change this block or the `repositories` block in the test module - it's not an _improvement_, it's a _tradeoff_ that I don't particularly think is worth it.

Reply:
The need for it is that as it is the convention, I assumed all I need to do to test a change is to reload the test module, which wasn't the case

Reply:
You didn't re-add the publishing block when you addressed the comment, but I think we're probably good otherwise